emsdk.py

```diff
@@ -1014,15 +1012,40 @@ def xcode_sdk_version():
     return subprocess.checkplatform.mac_ver()[0].split('.')
 
 
+def decide_cmake_target_platform(tool):
+  if hasattr(tool, 'arch') and tool.arch == 'aarch64':
+    return 'ARM64'
+  elif hasattr(tool, 'arch') and tool.arch == 'x86_64':
+    return 'x64'
+  elif hasattr(tool, 'arch') and tool.arch == 'x86':
+    return 'x86'
+  else:
+    if ARCH == 'aarch64':
+      return 'ARM64'
+    else:
+      return 'x64' if tool.bitness == 64 else 'x86'
+
+
+def decide_cmake_host_platform():
+  if ARCH == 'aarch64':
+    return 'ARM64'
+  elif ARCH == 'arm':
+    return 'ARM'
+  elif ARCH == 'x86_64':
+    return 'x64'
+  else:
+    return 'x86'
```

Comment:
How about writing this a dict so it will crash accordingly on unknown arch instead of defaulting to x86. e.g.


```
  cmake_arch = { 'aarch64': 'arm64' ... }
  return cmake_arch[ARCH]
```

Also should ARM64 and ARM be capitalized here.  I'd rather keep them all lowercase if that works?

Reply:
The code style fixed and the values changed according to the source reference.